### PR TITLE
Tekst inkl eksempler

### PR DESCRIPTION
**Pull request by Mikkelinski**
## 📥 Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## ☑️ Submission checklist

-   [ ] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [ ] `yarn build` works locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

## 💬 Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did, what alternatives you considered, etc...

### CODE REVIEW

File: packages/loader/README.md [hunk 1]
```diff
@@ -1,11 +1,9 @@
 # [`@fremtind/jkl-loader`](https://fremtind.github.io/jokul/components/loader/)
 
-## Om spinneren
+Lasteindikatoren viser at noe er underveis / at systemet jobber. Vi bruker den dersom en handling tar lengre tid enn to - tre sekunder.
```
Comment (Steinop) [inline packages/loader/README.md]: "Lasteindikatoren viser at systemet jobber og brukes hvis en handling tar lengre tid enn to – tre sekunder."

File: packages/loader/README.md [hunk 2]
```diff
@@ -1,11 +1,9 @@
 # [`@fremtind/jkl-loader`](https://fremtind.github.io/jokul/components/loader/)
 
-## Om spinneren
+Lasteindikatoren viser at noe er underveis / at systemet jobber. Vi bruker den dersom en handling tar lengre tid enn to - tre sekunder. 
 
-Vi bruker innlastingsspinneren for å vise at noe er underveis. Det kan være at systemet henter data, brukeren blir sendt til et annet sted eller det foregår beregninger. Vi bruker den når handlingen tar mer en tre sekunder.
+Det er viktig å bruke en lasteindikator for å trygge brukeren på at det faktisk skjer noe i bakkant. Lasteindikatoren kan ha ullik posisjon eller størrelse og skal settes i kontekst av det som trigget den.
```
Comment (Steinop) [inline packages/loader/README.md]: "… ha ulik posisjon…"

File: packages/loader/README.md [hunk 3]
```diff
@@ -1,11 +1,9 @@
...
-Vi bruker innlastingsspinneren for å vise at noe er underveis. Det kan være at systemet henter data, brukeren blir sendt til et annet sted eller det foregår beregninger. Vi bruker den når handlingen tar mer en tre sekunder.
+Det er viktig å bruke en lasteindikator for å trygge brukeren på at det faktisk skjer noe i bakkant. Lasteindikatoren kan ha ullik posisjon eller størrelse og skal settes i kontekst av det som trigget den. 
 
-Spinneren er et viktig designelement. Selv om det ikke er innlysende for brukeren hva som skjer i bakkant, kan vi trygge brukeren på at det faktisk skjer noe og vise det tydelig.
-
-### Størrelse
-
-Når vi vil vise at innlasting gjelder et bestemt element på en side eller et bestemt område, kan vi krympe spinneren, for eksempel slik at den passer på en knapp.
+### Eksempler
+- i en knapp  brukeren har trykket på
```
Comment (Steinop) [inline packages/loader/README.md]: "I en knapp …"

File: packages/loader/README.md [hunk 4]
```diff
@@ -1,11 +1,9 @@
...
+- i et kort eller felt hvor det skjer en beregning før innhold kan vises
```
Comment (Steinop) [inline packages/loader/README.md]: "I et kort …"

File: packages/loader/README.md (hunk 1 above)
Comment (Mikkelinski) [reply packages/loader/README.md]: oppdatert

**Pull request merged**